src/scss/tom-select.scss

```
@@ -81,8 +81,8 @@ $select-spinner-border-color:					$select-color-border !default;
 @import "../plugins/remove_button/plugin.scss";
 
 :root {
-	--ts-pr-clear-button: 0;
-	--ts-pr-caret: 0;
+	--ts-pr-clear-button: 0em;
+	--ts-pr-caret: 0rem;
```

Is there any reason two different units are used here? Of course, it shouldn't matter much, as 0 of anything is 0 of anything else.

---

> This PR adds the same unit used when the variable is not zero.

As it could be whatever unit this seemed a good baseline to me.

---

Not a blocker, it just looks strange to me. I'll just make these `px` so we can get this merged.

```suggestion
	--ts-pr-clear-button: 0px;
	--ts-pr-caret: 0px;
```